packages/uswds/src/Templates/ArrayFieldDescriptionTemplate.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
We don't need this line if you use the `tsconfig.json` with `compilerOptions: { "jsx": "react-jsx" }` set. Do copy one of the other `tsconfig.json` files.